pom.xml

```diff
@@ -730,6 +730,36 @@
             </activation>
             <build>
                 <plugins>
+                    <plugin>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <version>3.0.2</version>
```

Comment:
There is a property for the version ${plugins.maven.resources.version}

Reply:
ok thanks